Bug 71298

priestlg opened the pull request:
<!-- DO NOT DELETE THIS TEMPLATE -->

## Description
<!--
Full authoring Documentation pass
-->

---

This checklist is used to make sure that common guidelines for a pull request are followed. You can find a more complete discussion of PowerShell cmdlet best practices [here](https://msdn.microsoft.com/en-us/library/dd878270(v=vs.85).aspx).

- [x] **I have read the [contribution guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md).**
- [x] **If changes were made to any cmdlet, the XML help was regenerated using the [platyPSHelp module](https://github.com/Azure/azure-powershell/blob/dev/documentation/platyPSHelp-documentation.md).**
- [x] **If any large changes are made to a service, they are reflected in the respective [change log](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#updating-the-change-log).**

### [General Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#general-guidelines)
- [x] Title of the pull request is clear and informative.
- [x] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
- [x] The pull request does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/dev/documentation/breaking-changes.md) (unless a major version change occurs in the assembly and module).

### [Testing Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#testing-guidelines)
- [x] Pull request includes test coverage for the included changes.
- [x] PowerShell scripts used in tests should do any necessary setup as part of the test or suite setup, and should not use hard-coded values for locations or existing resources.

### [Cmdlet Signature Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#cmdlet-signature-guidelines)
- [x] New cmdlets that make changes or have side effects should implement `ShouldProcess` and have `SupportShouldProcess=true` specified in the cmdlet attribute. You can find more information on `ShouldProcess` [here](https://gist.github.com/markcowl/338e16fe5c8bbf195aff9f8af0db585d#what-is-the-change).
- [x] Cmdlet specifies `OutputType` attribute if any output is produced - if the cmdlet produces no output, it should implement a `PassThru` parameter.

### [Cmdlet Parameter Guidelines](https://github.com/Azure/azure-powershell/blob/dev/CONTRIBUTING.md#cmdlet-parameter-guidelines)
- [x] Parameter types should not expose types from the management library - complex parameter types should be defined in the module.
- [x] Complex parameter types are discouraged - a parameter type should be simple types as often as possible. If complex types are used, they should be shallow and easily creatable from a constructor or another cmdlet.
- [x] Cmdlet parameter sets should be mutually exclusive - each parameter set must have at least one mandatory parameter not in other parameter sets.

msftclas commented:
@priestlg,
Thanks for your contribution as a Microsoft full-time employee or intern. You do not need to sign a CLA.
_Thanks,
Microsoft Pull Request Bot_

azuresdkci commented:
Can one of the admins verify this patch?

priestlg commented:
PlatyPS was executed prior to the PR.

cormacpayne commented:
@priestlg Hey Graham, would you mind giving us a little more insight as to what the bug is that you are trying to fix with this pull request? I want to understand why these small changes to the examples are needed.

priestlg commented:
I'm getting this topics up to standards. For example, with this particular edits there needs to be a space between PS C:\> and the code.

priestlg commented:
The bug I'm working on is to "Standardize and normalize any new cmdlets.  Please lightly improve if you see something near the latest updates."

markcowl commented:
@priestlg I don't understand why this space is necessary.  If this is a standard that needs to be followed for some reason with examples, please explain why thsi is important, and lets make this part of the help generation and validation process

cormacpayne commented:
@priestlg Hey Graham, would you mind responding to the feedback from Mark? We plan on closing the pull request by the end of the week if there is no activity.

priestlg commented:
@cormacpayne, the previous emails got lost in my slew of Github emails and I apologize. The space is important because that has been a required standard of the PowerShell Documentation team. The extra space makes the code more legible. Please contact v-tmeyer or v-tamif for more information. Thanks.

TaraMeyer commented:
@cormacpayne  and @markcowl - this space also mimics the exact same format as what appears in the console.  If you open up a PowerShell session - you will see the "PS C:>" with a space automatically inserted.

cormacpayne commented:
@azuresdkci add to whitelist

cormacpayne commented:
@priestlg @TaraMeyer is this a change we want to make for _all_ cmdlets? This PR seems to just be taking care of Compute cmdlets

markcowl commented on src/ResourceManager/Compute/Commands.Compute/help/Remove-AzureRmVMBackup.md:
````diff
@@ -22,7 +22,7 @@ Remove-AzureRmVMBackup [-ResourceGroupName] <String> [-VMName] <String> [-Tag] <
 
 ### 1:
 ```
-PS C:\>
````
This looks like no example whatsover, why aren't we removing the example, or better yet, adding an example?

markcowl commented on src/ResourceManager/Compute/Commands.Compute/help/New-AzureRmVmss.md:
````diff
@@ -25,7 +25,7 @@ This cmdlet takes a **VirtualMachineScaleSet** object as input.
 ### Example 1: Create a VMSS
 ```
-PS C:\># Common
+PS C:\> # Common
````
It looks liek this example doesn't use prompts except on this line.  If we're going to touch an example, let's actually clean it up

priestlg replied:
Hi Mark, I'm used to doing multiple PRs when I'm actively working on Docs, so I normally do a find and replace on the prompt spacing then do a PR. I usually do multiple PRs to avoid merge conflicts. The fix to New-AzureRmVmss.md will be fixed in my next PR which will be later today.

markcowl commented:
on demand run here: http://azuresdkci.cloudapp.net/view/1-AzurePowerShell/job/powershell-demand/1696/

priestlg commented:
Thanks Mark, I fixed the other issue. Let me know when this is approved, because I have another PR coming down the line. I'll be sure to comment on everything I've updated too.

cormacpayne commented:
@priestlg this PR has been merged
